assessments/0003-krator.md

```diff
@@ -0,0 +1,181 @@
+
+## How this document works
+
+The assessment is divided into three sections: 
+
+- **Project documentation:** for end users of the project; aimed at people who intend to use it
+- **Contributor documentation:** for new and existing contributors to the project
+- **Website:** branding, website structure, and maintainability
+
+Within each section you receive a rating on different criteria. The full definition of each criteria is defined in [the criteria](criteria.md). We recommend reading each criterion’s definition.
+
+## Project documentation 
+
+
+| Criteria                   | 1    | 2   | 3   | 4   | 5   |
+| ---                        | ---  | --- | --- | --- | --- |
+| Information architecture   |  ✅  |     |     |     |     |
+| New user content           |  ✅  |     |     |     |     |
+| Content maintainability    |  ✅  |     |     |     |     |
+| Content creation processes |  ✅  |     |     |     |     |
+
+ Criteria: 
+ - 1 = (Is not present or requires significant work) 
+ - 3 = (is present, but needs work) 
+ - 5 = (is executed extremely well or no improvement required)
+
+### Comments
+
+- **Information Architecture**: 
+    - Documentation is currently in several locations and will need to be brought into one repo. The current resources are:
+        - The project [README](https://github.com/krator-rs/krator)
+        - [Introduction to krator Blog Post](https://deislabs.io/posts/introducing-krator/)
+        - [A Fistful of States: More State Machine Patterns in Rust](https://deislabs.io/posts/a-fistful-of-states/)
+        - [Crate krator API Documentation](https://docs.rs/krator/0.4.0/krator/). This is a substantial part of the current documentation and could be incorporated into the main body (site/repo) of the documentation. This is already a great resource for users, but if we transport it to a documentation repo, it will take some work if it's not already in markdown. Is there a way extract/download these guides quickly?
+        - [GitHub Documentation](https://github.com/krator-rs/krator/tree/main/docs). The Community guides teach contributing and development. 
+            - The [Developer Guide](https://github.com/krator-rs/krator/blob/main/docs/community/developers.md) covers prerequisites and building, which would make a good introductory or Getting Started entry point.
+            - The [Release Checklist](https://github.com/krator-rs/krator/blob/main/docs/community/release-checklist.md) appears aimed at developers of Krator, as opposed to first-time users. This should ideally be separated from the entry point.
+
+- **New user content**:
+    - We'll need to create a clear entry point for the new user. Some of this info could be taken from the [Developer Guide](https://github.com/krator-rs/krator/blob/main/docs/community/developers.md).
```

Comment:
Any thoughts/guidance on what content needs to be in the entry point/getting started guide? It would be great to get more specific with them.